src/docfx/lib/UrlUtility.cs

```diff
@@ -344,6 +344,30 @@ public static string RemoveLeadingHostName(string url, string hostName, bool rem
             : $"/{path}";
     }
 
+    public static string GetBookmark(string uid)
+    {
+        var sb = new StringBuilder(uid.ToLowerInvariant());
```

Comment:
What is the point of using a `StringBuilder` if everything else is string and regex replacement?

Reply:
Fixed.